libs/assertion/include/hpx/assertion/current_function.hpp

```diff
@@ -14,7 +14,7 @@
 
 namespace hpx { namespace assertion { namespace detail {
 
-    constexpr void current_function_helper()
+    void current_function_helper()
     {
```

Comment:
We have `HPX_CXX14_CONSTEXPR` for these cases.

Reply:
Doh, you're right. Will change.

Reply:
Done.